db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20181203235115) do
+ActiveRecord::Schema.define(version: 2019_03_26_173009) do
```

Comment:
hmm, not sure I want this update - perhaps we should delete schema.rb? as we've done in websiteone

Reply:
maybe... we certainly haven't had any negative side effects that I know of